src/org/openj9/envInfo/MachineInfo.java

```diff
@@ -175,12 +176,11 @@ private void validateInfo() {
 		boolean valid = true;
 		for (Info info : infoMap.values()) {
 			if (info.req != null) {
-				String version = parseInfo(info.output);
-				valid &= validateVersion(info.name, version, info.req);
+				valid &= validateVersion(info, info.req);
 			}
 		}
 		if (!valid) {
-			System.exit(1);
+//			System.exit(1);
```

Comment:
remove commented out code?

Reply:
This is on purpose. As described in the commit msg, we want to switch envDetector to warning mode. We will later remove the comment and add an option to toggle the failure or warning mode. I have added an comment on the code, so it is clearer.